### ISSUE TITLE
Have you any example to how use this?

### ISSUE THREAD

**Issue by josb86** [Golly-Rui/STM32_DMP_Driver#4]
Have you any example to how use this?

**Comment by fMeow** [comment]
Yes and no. Notice there is exactly two different version of DMP driver. One with a complete example and the other without.

For the one in master branch, there is no example.  But this version is simple and straight forward.
Just initialize by [`MPU6050_initialize()`](https://github.com/fMeow/STM32_DMP_Driver/blob/45e2ae39ce939808dab222be4fee2b3874f5e877/Source/MPU6050/mpu6050.c#L238) and [`DMP_Init()`](https://github.com/fMeow/STM32_DMP_Driver/blob/45e2ae39ce939808dab222be4fee2b3874f5e877/Source/MPU6050/mpu6050.c#L253). Then use [`Read_DMP()`](https://github.com/fMeow/STM32_DMP_Driver/blob/45e2ae39ce939808dab222be4fee2b3874f5e877/Source/MPU6050/mpu6050.c#L287) to update quaternion computed by DMP hardware in MPU6050. At last, read these quoternion in [q0,q1,q2,q3](https://github.com/fMeow/STM32_DMP_Driver/blob/45e2ae39ce939808dab222be4fee2b3874f5e877/Source/MPU6050/mpu6050.c#L21). Well, this code really sucks that the computed quoternion is hidden inside `MPU6050.c`. 

If your MCU is powerful enough, I recommend the other version that is located in [`I2Cdev`](https://github.com/fMeow/STM32_DMP_Driver/tree/I2Cdev) branch.
As for "I2Cdev" branch, there is a [working example](https://github.com/fMeow/STM32_DMP_Driver/tree/I2Cdev/Examples/STM32CubeMX) using STM32CubeMX  on STM32F103C8T6.
